client/README.md

```diff
@@ -54,7 +58,11 @@ const socketUrl = "https://api.mainnet.hiro.so/";
 const socket = io(socketUrl, {
   transports: [ "websocket" ]
 });
-const sc = new stacks.StacksApiSocketClient(socket);
+const sc = new stacks.StacksApiSocketClient(socket, {
+  headers: {
+    'x-hiro-product': 'blockchain-api-client',
+  },
+});
```

Comment:
Same here, a second parameter isn't valid code: https://github.com/hirosystems/stacks-blockchain-api/blob/9dc72488bff40de008d4245172259185840fc670/client/src/socket-io/index.ts#L32-L37

While socket.io has an option that we could use called ["extraHeaders"](https://socket.io/docs/v4/client-options/#extraheaders), they are not actual _http request_ headers and thus will not show up in the metric dashboards. It may be possible in the socket.io server code to detect `extraHeaders` and inject them into the http request such that it is reported in the logs, but that would need some investigation and testing.